src/zjs_net.c

```diff
@@ -214,13 +214,11 @@ static void post_closed(void *handle)
             zjs_free(h);
         }
 
-        if (net) {
```

Comment:
No need to check if server_h exists?

Reply:
Yeah, I don't think so. It should always be set if a socket exists; so considered an 'invariant'.